notebooks/samples/optimizer/ai_platform_optimizer_conditional_parameters.ipynb

```diff
@@ -0,0 +1,902 @@
+{
```

Comment:
<p>nit 1: Can we split this cel into 2:</p><p> </p><p>Create a new one for:</p><p> </p><p># Update to your username</p><p>USER = '[user-id]' #@param {type: 'string'}</p><p> </p><p>STUDY_ID = '{}_study_{}'.format(USER, datetime.datetime.now().strftime('%Y%m%d_%H%M%S')) #@param {type: 'string'}</p><p>REGION = 'us-central1'</p><p> </p><p>nit 2: Add double line between functions, not just a single line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>